libcore/IconInfo.vala

```diff
@@ -121,8 +121,8 @@ public class Files.IconInfo : GLib.Object {
         }
     }
 
-    public static Files.IconInfo? get_generic_icon (int size, int scale) {
-        var generic_icon = new GLib.ThemedIcon ("text-x-generic");
+    public static Files.IconInfo get_generic_icon (int size, int scale) {
+        var generic_icon = new GLib.ThemedIcon ("text-x-generic"); // Assume theme always contains this?
```

Comment:
it is in the fd.o spec so we should be able to assume it exists https://specifications.freedesktop.org/icon-naming-spec/latest/#names

Reply:
Thanks, I'll remove that comment then.